src/Deck.js

```diff
@@ -0,0 +1,13 @@
+class Deck {
```

Comment:
I also creat a deck class and test file for deck, forgot to mention that in the pull!